.github/workflows/docker.yaml

```
@@ -108,9 +108,11 @@ jobs:
         with:
           context: .
           file: production/Dockerfile
-          push: true
+          push: false
```

TODO: Change before merge
```suggestion
          push: true
```